### ISSUE TITLE
macOS: Command to Trust Certificate

### ISSUE THREAD

**Issue by dmhendricks** [10up/wp-local-docker-v2#32]
Great project you have going here. I'm not sure if/how this would work, but it would be nice to have a command to make it easy to trust the local certificate on macOS, something like
```bash
sudo 10updocker trust mywebsite-test
```

LBF does this via their GUI, which is convenient. _Screenshot purely provided to illustrate what I'm referring to - obviously, wp-local-docker implementation would be via CLI, if it's even possible._

Related: #22 

![lbf-trust-cert](https://user-images.githubusercontent.com/8184361/59163873-e6ff5c80-8acb-11e9-8316-5693be573bae.png)

**Comment by joseffb-mla** [comment]
Ran into this script for generating a local wild card cert that may be helpful as well:

```
openssl req \
    -newkey rsa:2048 \
    -x509 \
    -nodes \
    -keyout ~/Desktop/mysite.test.key \
    -new \
    -out ~/Desktop/mysite.test.crt \
    -reqexts SAN \
    -extensions SAN \
    -config <(cat /System/Library/OpenSSL/openssl.cnf \
        <(printf '
[req]
default_bits = 2048
prompt = no
default_md = sha256
x509_extensions = v3_req
distinguished_name = dn

[dn]
C = JP
ST = Tokyo
L = Tokyo
O = MyDomain Inc.
OU = Technology Group
emailAddress = hello@mysite.test
CN = mysite.test

[v3_req]
subjectAltName = @alt_names

[SAN]
subjectAltName = @alt_names

[alt_names]
DNS.1 = *.mysite.test
DNS.2 = mysite.test
')) \
    -sha256 \
    -days 3650
```

**Comment by eugene-manuilov** [comment]
The next release `2.8.0` will contain a command to generate certificates for local environments and to install local CA in the system trust store. Stay tuned.

**Comment by eugene-manuilov** [comment]
It has been implemented and merged via #138. We can consider this ticket as done. (Currently can be tested by installing `@next` version).
